src/main/java/com/mcmoddev/mmdbot/modules/commands/bot/management/CmdRefreshScamLinks.java

```diff
@@ -0,0 +1,33 @@
+package com.mcmoddev.mmdbot.modules.commands.bot.management;
+
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.mcmoddev.mmdbot.modules.logging.misc.ScamDetector;
+
+/**
+ * Refreshes the {@link com.mcmoddev.mmdbot.modules.logging.misc.ScamDetector#SCAM_LINKS}
+ *
+ * @author matyrobbrt
+ */
+public class CmdRefreshScamLinks extends Command {
+
+    public CmdRefreshScamLinks() {
+        name = "refresh-scam-links";
+        aliases = new String[]{"refreshscamlinks"};
+        help = "Refreshes the scam links";
+        category = new Category("Management");
+        hidden = true;
+        guildOnly = false;
+        ownerCommand = true;
```

Comment:
Poking @KiriCattus on the topic of whether this command should be locked to the bot owner(s) specifically.

Reply:
Owner for now but if needed we can increase to moderators in the future :)